data_scrapers/alameda_county.py

```diff
@@ -144,12 +144,13 @@ def get_demographics(out:Dict) -> (Dict, List):
     # format query to get entry for Alameda County
     param_list = {'where': "Geography='Alameda County'", 'outFields': '*', 'outSR':4326, 'f':'json'}
     # get cases data
-    response = requests.get(demographics_cases, params=param_list)
+    response = requests.get(demographics_cases, params = param_list) # type: ignore
```

Comment:
It would be better if we could resolve this without bailing out on types. Some options that should work:

- Make `outSR` a string:

    ```py
    param_list = {'where': "Geography='Alameda County'", 'outFields': '*', 'outSR': '4326', 'f':'json'}
    ```

- Define `param_list` as a compatible type. Unfortunately the actual type definition is [absurdly complex](https://github.com/python/typeshed/blob/ad6cf3a314fbecc5df7244664bb245870bae5552/third_party/2and3/requests/api.pyi#L14-L24), so we shouldn’t duplicate it, but we could do:

    ```py
    param_list: Dict[str, Union[str, int]] = {'where': "Geography='Alameda County'", 'outFields': '*', 'outSR': 4326, 'f':'json'}
    ```

    (Since we have more than one kind of value, MyPy just throws up its hands and says “could be anything,” which is too broad for the actual definition, so it needs a little hint here.)